### DISCUSSION TITLE
I am able to invoke the iOS Simulator using the below code

### DISCUSSION THREAD

**Discussion by anurag2apr1991** [appium/appium#15504] [General]
Hi,

I am able to invoke the iOS Simulator using the below code

`AppiumDriver driver = new AndroidDriver(url, capsIOS);`
I am passing the iOS capabilities to AndroidDriver and getting the iOS Simulator launched

Code

`DesiredCapabilities capsIOS = new DesiredCapabilities();`
 ` capsIOS.setCapability(MobileCapabilityType.PLATFORM_NAME, "iOS");`
`  capsIOS.setCapability(MobileCapabilityType.DEVICE_NAME, "iPhone 12 Pro Max");`
 ` capsIOS.setCapability(MobileCapabilityType.AUTOMATION_NAME, "XCUITest");`
 ` capsIOS.setCapability(MobileCapabilityType.UDID, "3FEF9D39-7F9A-454F-8599-9AE62AA2CA7F");`

`  capsIOS.setCapability("simulatorStartupTimeout",180000);`

`  String appUrl = System.getProperty("user.dir") + File.separator + "src" + File.separator + "main" + File.separator + "resources" + ``File.separator + "UICatalog-iphonesimulator.app";`
 `capsIOS.setCapability(MobileCapabilityType.APP,appUrl);`
`//  capsIOS.setCapability("bundleId", "com.example.apple-samplecode.UICatalog");`
`  URL url = new URL("http://0.0.0.0:4723/wd/hub");`
`AppiumDriver driver = new AndroidDriver(url, capsIOS);`

**Comment by KazuCocoa** [comment]
Do you want to say client-side issue?
So, the client should raise an error if some capabilities are not for iOS then.

Please create an issue in the proper client repository.
The decision will depend on the client's design.
